src/ert/cli/main.py

```diff
@@ -134,6 +134,7 @@ def run_cli(args):
         raise ErtCliError(model.getFailMessage())
 
 
+# pylint: disable=too-many-arguments
```

Comment:
In general. i think that silencing these linter issues is wrong. Sure, some issues reported we already can say with certainty that they are false-positives, but by silencing others (or indeed all), like too-many-arguments (just as an example), we inhibit the linter expressing code-rot or sections that could/should be improved later on. We are left with no metric on how the quality of the codebase. I'd argue it is better to globally remove certain variants instead.
Just a thought.

Reply:
thank you for bringing it up! in general i agree with you : )

i also try to fix them wherever i can, which is why i was a bit sad to add the ignore here.

i think in this particular case, one could fix it easily - the arguments ensemble size and and current case name are both accessible from ert. but i also think (this is an assumption) that if i suggest to remove them, i will get the feedback that those were used to make it more explicit which data is needed rather than just passing around the whole ert / enkfmain object.

my guess would be that the right approach here is to ... collect the arguments that are needed to run the cli in a class, and pass that one?

do you think we should try and discuss this? see what's desirable / possible here?

Reply:
I think the approach you did is the correct one -- try to squash as many as can be done.
After that, there is always an open discussion on the level of value for individual issue variants.
Leave this as is, -- i just wanted to point out that the no-warn-by-any-means-approach is discouraged.

Some work you've done here. Well done! :+1: